src/main.c

```diff
@@ -17,7 +17,7 @@ static void print_help(void)
 {
     puts("Usage: " APP_NAME " [OPTION...] FILE...");
     puts("  -f, --fullscreen         Full screen mode");
-    puts("  -g, --geometry=X,Y,W,H   Set window geometry");
+    puts("  -g, --geometry=X,Y WxH   Set window geometry");
```

Comment:
It is not a common way to specify program arguments.
The "WxH" in this case is a next argument, it doesn't relate to geometry.
To fix this you have to escape a space or use quotes: `--geometry=X,Y\ WxH` or `--geometry="X,Y WxH"`.
Looks ugly for me =)

What about modifying `parse_rect()` to support both formats, or even better, to support any delimiters: commas, spaces, 'x', etc?